### ISSUE TITLE
php5-xdebug broken

### ISSUE THREAD

**Issue by MiteshShah** [gplessis/dotdeb-php5#83]
```
The following packages have unmet dependencies:
 php5-xdebug : Depends: phpapi-20100525
               Depends: php5-common (= 5.4.40-1~dotdeb+wheezy.1) but 5.6.8-1~dotdeb+wheezy.1 is to be installed
```

**Comment by gplessis** [comment]
I'm working on it : https://github.com/gplessis/dotdeb-xdebug

**Comment by MiteshShah** [comment]
@gplessis Thanks 
ping @gau1991

**Comment by gplessis** [comment]
`php5-xdebug` has finally been published for PHP 5.6
